web/src/containers/P2P/OrderWaitPayment/index.tsx

```diff
@@ -211,14 +230,20 @@ const OrderWaitPayment: FC<Props> = (props: Props): ReactElement => {
                         {translate('page.body.p2p.order.transfer.order.wait.confirm')}
                     </Button>
                     <Button
-                        onClick={() => window.console.log('dispute')}
+                        onClick={() => dispatch(p2pOrderUpdateStatus('dispute'))}
```

Comment:
you should clarify this with Maxim but as I know order receives state: `autocancelled`
and we need to comment in code 'Log dispute' button, because this functionality will be for the future. Discuss it with Maxim please